packages/compiler/src/core/projection-members.ts

```diff
@@ -405,7 +405,7 @@ export function createProjectionMembers(checker: Checker): {
       },
       toKebabCase(base) {
         return createFunctionType(() => {
-          return createLiteralType(paramCase(base.value));
+          return createLiteralType(kebabCase(base.value));
```

Comment:
Whoops 😂

Reply:
that was a breaking change on their side